Tests/Unit/DependencyInjection/ConfigurationTest.php

```diff
@@ -14,62 +14,79 @@
 use Symfony\Cmf\Bundle\ResourceBundle\DependencyInjection\CmfResourceExtension;
 use Symfony\Cmf\Bundle\ResourceBundle\DependencyInjection\Configuration;
 use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractExtensionConfigurationTestCase;
+use Symfony\Component\Config\Loader\Loader;
+use Symfony\Component\Config\Definition\Processor;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
 
-class ConfigurationTest extends AbstractExtensionConfigurationTestCase
```

Comment:
I removed the DI testing library -- in general I find it to abstract what is actually going on, which, infact, isn't that much.